Source/WebCore/animation/KeyframeInterpolation.cpp

```diff
@@ -0,0 +1,31 @@
+/*
```

Comment:
Why do you need this file?

Reply:
I don’t need it at this stage, it’s true. I can add it in the next patch.